### PR TITLE
Isolate nailgun

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ python:
 addons:
   apt:
     packages:
-    - rpm
+      - rpm
+  pip:
```
Comment (mdellweg) [inline .travis.yml]: Seems like travis ignores this. ;(

Comment (evgeni) [reply .travis.yml]: I don't think there is a `pip` addon?

Comment (mdellweg) [reply .travis.yml]: twas worth a try. I woulda expect it to fail, though

File: plugins/module_utils/ansible_nailgun_cement.py [hunk 2]
```diff
@@ -5,56 +5,110 @@
...
+
+
+class ForemanAnsibleModule(ForemanBaseAnsibleModule):
+
+    def check_requirements(self):
+        if not HAS_NAILGUN:
+            self.fail_json(msg='The nailgun Python module is required',
+                           exception=NAILGUN_IMP_ERR)
+
+    def connect(self, ping=True):
```
Comment (evgeni) [inline plugins/module_utils/ansible_nailgun_cement.py]: interestingly, this does not actually call `ping()` (and never did it seems).

File: tests/test_crud.py [hunk 3]
```diff
@@ -112,5 +112,4 @@ def run_playbook(module, extra_vars=None, limit=None):
 @pytest.mark.parametrize('module', MODULES)
 def test_crud(tmpdir, module, record):
     run = run_playbook_vcr(tmpdir, module, record=record)
-    print(run.stdout.read())
```
Comment (evgeni) [inline tests/test_crud.py]: this is not needed because because run playbook already will print stuff?

File: .gitignore [hunk 4]
```diff
@@ -7,4 +7,5 @@ __pycache__
 *.pyo
 *.swp
 .pytest_cache/
+.mypy_cache/
```
Comment (evgeni) [inline .gitignore]: this seems unrelated?

Comment (mdellweg) [reply .gitignore]: Yes, unrelated. Mypy was able to pinpoint syntaxerrors codestyle did not find.

Comment (mdellweg) [reply .gitignore]: I'd like to keep it anyways.

File: tests/test_crud.py (hunk 3 above)
Comment (mdellweg) [reply tests/test_crud.py]: Yes, i noticed the output (in case of a failure) was doubled.

**Comment by mdellweg** [comment]
In general, we could remove the Apypie from the module names now. Do we want to do it in this PR?

**Comment by evgeni** [comment]
I'd prefer not to right now.

File: .gitignore (hunk 4 above)
Comment (evgeni) [reply .gitignore]: fine with me
